.coveragerc

```diff
@@ -0,0 +1,48 @@
+[run]
+# Enable branch coverage measurement
+branch = True
+
+# Source files to measure
+source = airbrakes
+
+# Patterns of files to omit from coverage
+omit =
+    */tests/*
+    */site-packages/*
+    setup.py
+    airbrakes/mock/*
+
+[paths]
+source =
+    src/
+    */site-packages/
+
+[report]
+# Don't report files with 100% coverage
+skip_covered = False
+
+# Don't report files with no executable code
+skip_empty = True
+
+# Exclude certain lines from coverage measurement
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code
+    raise AssertionError
+    raise NotImplementedError
```

Comment:
NotImplementedError is only in display.py, and AssertionError is not used?

Reply:
boiler plate code, we might do it in the future idk